### PR TITLE
fix: add missing reveal password button

### PR DESCRIPTION
**Pull request by netbe**
----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [x] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [ ] contains a reference JIRA issue number like `SQPIT-764`
  - [ ] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [ ] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

### Issues

Adds missing reveal password button to login screens

### Solutions

* add cta action button and remove inline confirm button

### Dependencies (Optional)

_If there are some other pull requests related to this one (e.g. new releases of frameworks), specify them here._

Needs releases with:

- [ ] GitHub link to other pull request

#### Test Coverage (Optional)

- [ ] I have added automated test to this contribution

#### How to Test

_Briefly describe how this change was tested and if applicable the exact steps taken to verify that it works as expected._

### Notes (Optional)

_Specify here any other facts that you think are important for this issue._

### Attachments (Optional)

_Attachments like images, videos, etc. (drag and drop in the text box)_

----
#### PR Post Submission Checklist for internal contributors (Optional)

 - [ ] Wire's Github Workflow has automatically linked the PR to a JIRA issue
----
#### PR Post Merge Checklist for internal contributors

 - [ ] If any soft of configuration variable was introduced by this PR, it has been added to the relevant documents and the CI jobs have been updated.
----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

**Review by agisilaos — APPROVED** [review]
Looking good. Left a comment and let's try to fix the SwiftLint warnings before merging.

File: Wire-iOS/Sources/UserInterface/Components/Views/EmailPasswordTextField.swift [hunk 1]
```diff
@@ -31,6 +31,12 @@ extension EmailPasswordTextFieldDelegate {
 
 class RevisedEmailPasswordTextField: EmailPasswordTextField {
 
+    override func configureSubviews() {
+        super.configureSubviews()
+//        passwordField.bindConfirmationButton(to: emailField)
+//        passwordField.confirmButton.addTarget(self, action: #selector(confirmButtonTapped), for: .touchUpInside)
+    }
```
[the remark this answers is not in the document]
Comment (netbe) [reply Wire-iOS/Sources/UserInterface/Components/Views/EmailPasswordTextField.swift]: removed

**Review by johnxnguyen — APPROVED** [review]

File: Wire-iOS/Sources/UserInterface/Components/Views/EmailPasswordTextField.swift [hunk 2]
```diff
@@ -131,8 +129,7 @@ class EmailPasswordTextField: UIView, MagicTappable {
     }
 
     func configureConstraints() {
-        contentStack.translatesAutoresizingMaskIntoConstraints = false
-
+        [passwordField, emailField, contentStack].forEach { $0.translatesAutoresizingMaskIntoConstraints = false }
```
Comment (MarcinR) [inline Wire-iOS/Sources/UserInterface/Components/Views/EmailPasswordTextField.swift:132]: You can use [passwordField, emailField, contentStack].prepareForLayout()

File: Wire-iOS/Sources/UserInterface/Components/Views/EmailPasswordTextField.swift [hunk 3]
```diff
@@ -32,7 +32,7 @@ extension EmailPasswordTextFieldDelegate {
 class RevisedEmailPasswordTextField: EmailPasswordTextField {
 
     override func configureConstraints() {
-        contentStack.translatesAutoresizingMaskIntoConstraints = false
+        [passwordField, emailField, contentStack].forEach { $0.translatesAutoresizingMaskIntoConstraints = false }
```
Comment (MarcinR) [inline Wire-iOS/Sources/UserInterface/Components/Views/EmailPasswordTextField.swift:35]: You can use [passwordField, emailField, contentStack].prepareForLayout()

**Review by MarcinR — APPROVED** [review]

**Pull request merged**